### PR TITLE
Add tui.calender w/ git auto-update via single package.json

### PR DESCRIPTION
**Pull request by RohitChattopadhyay**
Pull request for issue: #12852

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):  https://github.com/nhnent/tui.calendar
 * Official website (optional, not the repository): https://ui.toast.com/tui-calendar/
 * NPM package url (optional): https://www.npmjs.com/package/tui-calendar
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 135
   - Count of stars: 5964
   - Count of forks: 303
 * NPM download stats (optional):
   - Downloads in the last day: 28
   - Downloads in the last week: 333
   - Downloads in the last month: 838

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] Doesn't already exist / has duplicate issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
 * [x] The parent of the commit(s) in the PR is not older than 3 days.
 * [x] Pull request is sent from a non-master branch with a meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/tui.calender/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "tui.calender",
```
Comment (sashberd) [inline ajax/libs/tui.calender/package.json]: Please fetch name according to https://github.com/nhnent/tui.calendar/blob/2a1d526f9491855dc83131334bc5fbf6f9c80560/package.json#L2

File: ajax/libs/tui.calender/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "filename": "tui-calender.min.js",
+  "description": "A full featured JavaScript schedule calendar.",
+  "keywords": [
```
Comment (sashberd) [inline ajax/libs/tui.calender/package.json]: Lets add another keywords from lib origin: https://github.com/nhnent/tui.calendar/blob/2a1d526f9491855dc83131334bc5fbf6f9c80560/package.json#L12

File: ajax/libs/tui.calender/package.json (hunk 1 above)
Comment (RohitChattopadhyay) [reply ajax/libs/tui.calender/package.json]: Hi @sashberd, thanks for the review, but the name of repository  is tui.calender and in the docs its specifies that name in package.json should match with repository name. Please suggest what to do
Thanks

Comment (sashberd) [reply ajax/libs/tui.calender/package.json]: Well in package.json of lib the name is : tui-calendar

Comment (RohitChattopadhyay) [reply ajax/libs/tui.calender/package.json]: I also had the same confusion hence I tried it using tui-calender ,but it failed in CI .You can see the report https://ci.cdnjs.com/cdnjs/cdnjs/21784

Also please suggest what keyword needs to be added
Thanks for help

**Pull request closed without merging**